app/src/main/java/com/babylon/wallet/android/presentation/settings/personas/PersonasViewModel.kt

```diff
@@ -51,7 +55,9 @@ class PersonasViewModel @Inject constructor(
                         babylonFactorSource = babylonFactorSource
                     )
                 }
-            }.collect {}
+            }
+                .flowOn(defaultDispatcher)
+                .collect {}
```

Comment:
To prevent unnecessary instantiating a FlowCollector, there is a `collect` method with no params.